docs/en/api/Java-API.md

```diff
@@ -92,6 +92,8 @@ idiomatic way to create files is to use
 [`FileSystem#createFile(AlluxioURI)`](https://docs.alluxio.io/os/javadoc/{{site.ALLUXIO_MAJOR_VERSION}}/alluxio/client/file/FileSystem.html#createFile-alluxio.AlluxioURI-),
 which returns a stream object that can be used to write the file. For example:
 
+>Note: there are some file names limitation when creating file through Alluxio, please check : [Alluxio limitation on file names](https://docs.alluxio.io/ee/user/edge/en/operation/Troubleshooting.html)
```

Comment:
```suggestion
> Note: there are some file name limitation when creating files through Alluxio. Please check [Alluxio limitation on file names]({{ '/en/operation/Troubleshooting.html' | relativize_url }}#alluxio-limitation-on-file-names)
```

for references to other doc pages, we have to dynamically construct the url to match with the corresponding version and edition that the page is built for. the syntax above invokes a functional call to do so

since the new section has a header, you can suffix the url with an anchor to load the page at the desired header via `#my-subheader-title`

Reply:
thanks I intended to do that but don't know how